akka-actor-typed/src/main/scala/akka/actor/typed/ActorRef.scala

```diff
@@ -20,7 +21,10 @@ import akka.actor.typed.internal.InternalRecipientRef
  * messages are delivered to the [[DeadLetter]] channel of the
  * [[akka.event.EventStream]] on a best effort basis
  * (i.e. this delivery is not reliable).
+ *
+ * Not for user extension
```

Comment:
Internal use only.

Reply:
"Not for user extension" is the phrase we have used in other places. It's still public user api, but we don't support external subclasses

Reply:
Thanks for the clarification.